src/datepicker/datepicker.js

```diff
@@ -571,6 +579,8 @@ function(scope, element, attrs, $compile, $parse, $document, $rootScope, $positi
     if (datepickerPopupConfig.html5Types[attrs.type]) {
       dateFormat = datepickerPopupConfig.html5Types[attrs.type];
       isHtml5DateInput = true;
+      ngModelOptions.timezoneHtml5 = ngModelOptions.timezone;
```

Comment:
This is a great question. Looking through the changes, the distinction seems superfluous. I forget why I made the distinction in the first place. Let me remove the distinction.